sympy/algebras/quaternion.py

```diff
@@ -954,13 +954,13 @@ def coplanar(cls, q1, q2):
         aq = q1.axis()
         ar = q2.axis()
 
-        return (aq == ar or aq == -ar)
+        return (aq - ar).is_zero_quaternion() or (aq + ar).is_zero_quaternion()
```

Comment:
This should be `fuzzy_or` because otherwise `None or False` gives `False`:
```python
In [20]: print(fuzzy_or([None, False]))
None

In [21]: print(None or False)
False
```